test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -57,6 +57,9 @@
     ("aten::_linalg_qr_helper", datetime.date(2022, 8, 1)),
     ("aten::linalg_lu_solve", datetime.date(2022, 8, 1)),
     ("aten::linalg_lu_solve.out", datetime.date(2022, 8, 1)),
+    ("aten::linalg_det", datetime.date(2022, 8, 1)),
+    ("aten::linalg_det.out", datetime.date(2022, 8, 1)),
```

Comment:
2Yes, It is because of the change of the input. When I rewrote the linalg docs, I changed the name of the input in the docs to be `A`, `B`... and Mike mentioned that we could change the name in the future (read "not for this PR" :P). I think these ports to structured kernels are a good time to change it.